### ISSUE TITLE
where are the backup{1,2,3,4}  come from

### ISSUE THREAD

**Issue by cn00** [docker-library/redis#378]
![WeChat8f177cfa660e68e4c9c46d9e0cdb6a0d](https://github.com/docker-library/redis/assets/5587043/04675673-938c-4921-ad22-7c51e96f5a8f)
[45.83.123.29--cleanfda--init.sh.txt](https://github.com/docker-library/redis/files/12809095/45.83.123.29--cleanfda--init.sh.txt)

I use the latest image sha256:7c4b517da47d331a47827390b9e8eb1be7ee68133af9c332660001b4d447828d

**Comment by LaurentGoderre** [comment]
I am not finding a matching image with that digest. What tag did you use?

**Comment by yosifkit** [comment]
If you or your application did not create those values, then that looks like an external malicious actor. See also the [Security section](https://github.com/docker-library/docs/tree/e6bbe741f0bbcf9185ebec7f924d608bc777e196/redis#security) from the Docker Hub docs.

TL;DR: if you expose your `redis` container instance to the internet, it is trivial to "hack" it.
